src/templates/core/proposals/add_editors.html

```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
 {% load static from staticfiles %}
 {% load bootstrap3 %}
+{% load ascii_encode %}
```

Comment:
Exhibit A for use of ```ascii_encode``` which is why I'm suggesting you change it so that it converts unicode characters to html entities, rather than remove them completely